### ISSUE TITLE
Error in using the pretrained model

### ISSUE THREAD

**Issue by hdubey** [santi-pdp/segan#61]
Hi,
I tried to use the pre-trained model w/ biasing. I got the following error:

DataLossError (see above for traceback): Unable to open table file segan_results/SEGAN-41700.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
